core-common/src/main/java/org/zalando/nakadi/domain/BatchItem.java

```diff
@@ -63,6 +68,7 @@ public static EmptyInjectionConfiguration build(final int position, final boolea
     private String brokerId;
     private String eventKey;
     private int eventSize;
+    private EventOwnerHeader owner;
```

Comment:
`owner` is not the owner, it is EventOwnerHeader

Reply:
I don't get the point. Class name is `EventOwnerHeader`, logical meaning is owner. 
I mean i'm not limiting myself to call all variables of class `String` to `string` only

Reply:
then please rename the class name to represent logical meaning :)